js/src/interface.js

```diff
@@ -119,11 +119,15 @@ let app = new Vue({
     manageDataBtn: false
   },
   computed: {
+    checkPackage() {
+      if (data.id === 'com.fliplet.data-source-query') return true;
```

Comment:
```suggestion
      return data.id === 'com.fliplet.data-source-query';
```

Also, I don't understand the logic of this computed variable. Why is it checking for `data.id`?